ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -2143,22 +2145,22 @@ periodics:
       command:
       - /osde2e
       env:
-      - name: ADDON_CLEANUP_HARNESSES
-        value: quay.io/integreatly/integreatly-operator-cleanup-harness:osde2e
       - name: ADDON_IDS
         value: rhmi
       - name: ADDON_POLLING_TIMEOUT
         value: "7400"
-      - name: ADDON_RUN_CLEANUP
-        value: "true"
```

Comment:
since we no longer test with enabled AWS storage in osde2e, it's not required to use the "cleanup" image anymore